test/unittests/skills/core.py

```diff
@@ -58,20 +58,25 @@ class MycroftSkillTest(unittest.TestCase):
     def setUp(self):
         self.emitter.reset()
 
-    def check_vocab_from_file(self, filename, vocab_type=None, result_list=[]):
+    def check_vocab_from_file(self, filename, vocab_type=None, result_list=None
+                              ):
```

Comment:
This looks a little funny. Maybe we should move the last item down, as well so that it's `result_list=None):`.